dist/tools/pyterm/pyterm

```diff
@@ -891,6 +931,8 @@ if __name__ == "__main__":
                         action="store_false",
                         help="Do not try to reconnect when failing on "
                              "connection setup (Default)")
+    parser.add_argument("process_tap", nargs='*')
```

Comment:
Does this still make sense? Why isn't this passed together with `-pa`?

Reply:
Hm, good question. I thought native was expecting the tap parameter as last parameter but apparently it does also work in between. So, yes, could be probably removed.

Reply:
I think it will then ignore the parameters that come after

Reply:
Nope, all are handled, e.g., using `gnrc_networking` with `USE_ZEP=1` and `USEMODULES += netdev_tap`
```
bin/native/gnrc_networking.elf  -z '[::1]:17754' tap0 -UAB
RIOT native interrupts/signals initialized.
RIOT native board initialized.
RIOT native hardware initialization complete.
...
ifconfig
...
Iface  8  HWaddr: 00:00  Channel: 26  NID: 0x23  PHY: O-QPSK 
          Long HWaddr: AB:00:00:00:00:00:00:00 
```
I.e., -z and -U are handled correctly.

Reply:
And even if: Since the make system provides the parameters in most cases, any constraints on them can be ensured there.